### PR TITLE
Miscellaneous history fixes and additions

### PR DESCRIPTION
**Pull request by loup99**
Opening this pull-request to keep track:
- Fixed dead Ostrogothic ruler holding a series of counties in southern Gaul
- Fixed the county of Porto not having any history after the death 35
- Fixed the bug with the Bureaucratic government retinue bonus not added in the retinue total
- Changed the Lombard duchy colours in Rule of Dukes bookmarks to make them more distinct
- Added two new dynamic story events around the Isaurian Wars
- Fixed bug that caused the Exilarchate to disband as Exilarchs have to have "House of David" trait
- Moved Isaurian characters to isaurian.txt
- Added a few Isaurian generals
- Added some character info links
- Added historical war of usurper Leontius
- Placeholder localisation for syn.10.a
- Fixed the Western Roman Imperial regalia
- Changed the names of some historical Popes
- Added a dynasty for Samaritan High Priests without a dynasty
- Additional Objectives localisation fix
- Fixed the culture of St. Andrew

### CODE REVIEW

File: WTWSMS/history/characters/aramean.txt [hunk 1]
```diff
@@ -24,12 +24,14 @@
 	}
 }
 154000 = {
-	name="Le�ntios"
+	name="Leontios"
```
Comment (loup99) [inline WTWSMS/history/characters/aramean.txt]: WAD spelling, not a typo

File: WTWSMS/history/characters/isaurian.txt [hunk 2]
```diff
@@ -1,5 +1,5 @@
 154001 = {
-	name="Longinius"
+	name="Longinus"
```
Comment (loup99) [inline WTWSMS/history/characters/isaurian.txt]: Latin vs Anglicised, no?

File: WTWSMS/history/provinces/746 - Ephesos.txt [hunk 3]
```diff
@@ -18,7 +18,7 @@ b_lebedos = temple # Orthodox Holy Site
 
 # Misc
 culture = greek
-religion = hellenic_pagan
+religion = nicene
```
Comment (loup99) [inline WTWSMS/history/provinces/746 - Ephesos.txt]: As discussed.

File: WTWSMS/localisation/AO_events.csv [hunk 4]
```diff
@@ -187,7 +187,7 @@ ruthless_desc;This character believes that the ends justify the means, and will
 honorable;Honorable;Honorable;;;;;;;;;;;;x
 honorable_desc;This character is a firm believer in the importance of duty, mercy, and honor.;Ce personnage croit fermement en l'importance du devoir, de la piti� et de l'honneur.;;;;;;;;;;;;x
 plot_kidnap_character_title;Kidnap [This.GetTitledFirstName];Kidnapper [This.GetTitledFirstName];[This.GetTitledFirstName] entf�hren;;;;;;;;;;;x
-plot_kidnap_character_short;Kidnap [This.GetTitledFirstName];Kidnapper [This.GetTitledFirstName];;;;;;;;;;;;x
+plot_kidnap_character_short;Plot to Kidnap;Kidnapper [This.GetTitledFirstName];;;;;;;;;;;;x
```
Comment (loup99) [inline WTWSMS/localisation/AO_events.csv]: Why did you change this?

File: WTWSMS/history/characters/isaurian.txt (hunk 2 above)
Comment (IhateTrains) [reply WTWSMS/history/characters/isaurian.txt]: Latin form is "Longinus" and "Longinus" is used in most(all?) Egnlish sites

File: WTWSMS/localisation/AO_events.csv (hunk 4 above)
Comment (IhateTrains) [reply WTWSMS/localisation/AO_events.csv]: It's broken, returns your character instead of target

Comment (loup99) [reply WTWSMS/localisation/AO_events.csv]: Should be FROM as the scope, no?

File: WTWSMS/history/characters/isaurian.txt (hunk 2 above)
Comment (loup99) [reply WTWSMS/history/characters/isaurian.txt]: While both Longinius and Longinus are Latin names, I think in English Longinus is preferred. Yet it doesn't matter here.

File: WTWSMS/localisation/AO_events.csv (hunk 4 above)
Comment (IhateTrains) [reply WTWSMS/localisation/AO_events.csv]: I think I tested with FROM and it didn't work.

Comment (loup99) [reply WTWSMS/localisation/AO_events.csv]: What happens with [From.GetTitledFirstName]?

File: WTWSMS/history/characters/aramean.txt (hunk 1 above)
Comment (IhateTrains) [reply WTWSMS/history/characters/aramean.txt]: Reverted. Are you sure this displays properly in game?

Comment (loup99) [reply WTWSMS/history/characters/aramean.txt]: Not with your change.

File: WTWSMS/history/provinces/746 - Ephesos.txt (hunk 3 above)
Comment (IhateTrains) [reply WTWSMS/history/provinces/746 - Ephesos.txt]: Done...

File: WTWSMS/history/characters/aramean.txt (hunk 1 above)
Comment (IhateTrains) [reply WTWSMS/history/characters/aramean.txt]: Could you revert it yourself? I copied from the master branch and somehow it's still different.

File: WTWSMS/localisation/AO_events.csv (hunk 4 above)
Comment (IhateTrains) [reply WTWSMS/localisation/AO_events.csv]: That is what I used, From not FROM

File: WTWSMS/history/characters/aramean.txt (hunk 1 above)
Comment (loup99) [reply WTWSMS/history/characters/aramean.txt]: Yes, but copied via GitHub? That never works, just for the information.

Comment (IhateTrains) [reply WTWSMS/history/characters/aramean.txt]: No, via Notepad++

File: WTWSMS/history/characters/hindustani.txt [hunk 5]
```diff
@@ -218,7 +218,7 @@
 	}
 }
 190205 = {
-	name="Eusabic" # Theodoric's Mother
+	name="Ereleuva" # Theodoric's Mother
```
Comment (loup99) [inline WTWSMS/history/characters/hindustani.txt]: She was baptised in the name of Eusebia, which is Eusabic in Gothic if I get it correctly.

Comment (IhateTrains) [reply WTWSMS/history/characters/hindustani.txt]: I found that too, but didn't find the Gothic form. Now that I know what language it is, I confirm that Eusabic is a good form. I will revert it.

Comment (IhateTrains) [reply WTWSMS/history/characters/hindustani.txt]: Done.

File: WTWSMS/events/WtWSMS_story_events.txt [hunk 6]
```diff
@@ -3599,4 +3598,112 @@ narrative_event = {
...
+		NOT = { has_global_flag = isaurian_war_ended }
+	}
+	
+	mean_time_to_happen = {
+		days = 1
+	}
+	
+	immediate = {
+		set_global_flag = isaurian_war_ended
+		d_isauria = { holder_scope = { save_global_event_target_as = target_isaurian_leader } }
```
Comment (loup99) [inline WTWSMS/events/WtWSMS_story_events.txt]: Why do you set a global_event_target?

File: WTWSMS/events/WtWSMS_story_events.txt [hunk 7]
```diff
@@ -3599,4 +3598,112 @@ narrative_event = {
...
```
Comment (loup99) [inline WTWSMS/events/WtWSMS_story_events.txt]: Same as above, the one receiving the event is the holder of d_isauria.

File: WTWSMS/history/characters/hebrew.txt [hunk 8]
```diff
@@ -11,7 +11,7 @@
 	}
 }
 
-# IDs above 10 digits cause problems
+# IDs above 10 digits may cause problems
```
Comment (loup99) [inline WTWSMS/history/characters/hebrew.txt]: We know they cause problems, the most notable of which are crashes.

File: WTWSMS/localisation/WtWSMS_events.csv [hunk 9]
```diff
@@ -1138,7 +1138,7 @@ imperial_legacy.49.name;[From.PrimaryTitle.GetName] disbanding;[From.PrimaryTitl
...
+historical_wtwsms.1.desc;You are now once again the emperor in Constantinople. In 475, you had after a conspiracy lost the throne to the Miaphysite Basiliscus and been forced to flee the capital. Subsequently after loosing the throne and being defeated, Basiliscus has been imprisoned and safely awaits your decision in prison...;Vous �tes maintenant de nouveau l'empereur de Constantinople. En 475, apr�s un complot, vous aviez perdu le tr�ne au profit du Miaphysite Basiliscus et aviez �t� contraint de fuir la capitale. Apr�s avoir �t� vaincu et puis perdu le tr�ne, Basiliscus a d�sormais �t� emprisonn� et attend en toute s�curit� votre d�cision en prison...;;;;;;;;;;x
```
Comment (loup99) [inline WTWSMS/localisation/WtWSMS_events.csv]: Any reason for jail -> prison? With prison and imprisoned the text is a bit repetitive, hence my usage of jail.

File: WTWSMS/localisation/AO_events.csv (hunk 4 above)
Comment (loup99) [inline WTWSMS/localisation/AO_events.csv]: The title works, but not the short?

File: WTWSMS/events/WtWSMS_story_events.txt (hunk 6 above)
Comment (IhateTrains) [reply WTWSMS/events/WtWSMS_story_events.txt]: I did that be be sure it worked :) Is normal event_target enough here (for the localisation)?

File: WTWSMS/events/WtWSMS_story_events.txt (hunk 7 above)
Comment (IhateTrains) [reply WTWSMS/events/WtWSMS_story_events.txt]: OK, I will simplyfy this

File: WTWSMS/events/WtWSMS_story_events.txt (hunk 6 above)
Comment (loup99) [reply WTWSMS/events/WtWSMS_story_events.txt]: Definitely. And below you don't need it at all, as it isn't called and the character receiving the event is your target.

File: WTWSMS/history/characters/hebrew.txt (hunk 8 above)
Comment (IhateTrains) [reply WTWSMS/history/characters/hebrew.txt]: But not all 10 digit character cause problems IIRC. Only characters with IDs above integer limit (2147483647) should be problematic.

Comment (loup99) [reply WTWSMS/history/characters/hebrew.txt]: I would prefer to not risk it...

File: WTWSMS/localisation/WtWSMS_events.csv (hunk 9 above)
Comment (IhateTrains) [reply WTWSMS/localisation/WtWSMS_events.csv]: "Prison" seems more "medieval" :P but that was my personal impression, I can revert it.

File: WTWSMS/localisation/AO_events.csv (hunk 4 above)
Comment (IhateTrains) [reply WTWSMS/localisation/AO_events.csv]: Title is what appears in the plot screen?

File: WTWSMS/localisation/WtWSMS_events.csv (hunk 9 above)
Comment (loup99) [reply WTWSMS/localisation/WtWSMS_events.csv]: No, we can use another synonym in that case. "dungeon" or "penitentiary" could be alternatives.

File: WTWSMS/localisation/AO_events.csv (hunk 4 above)
Comment (loup99) [reply WTWSMS/localisation/AO_events.csv]: Which one is it that works?

File: WTWSMS/events/WtWSMS_story_events.txt (hunk 7 above)
Comment (IhateTrains) [reply WTWSMS/events/WtWSMS_story_events.txt]: Done.

Comment (loup99) [reply WTWSMS/events/WtWSMS_story_events.txt]: You don't need this at all, actually. It isn't used in the localisation for that event,

File: WTWSMS/localisation/AO_events.csv (hunk 4 above)
Comment (IhateTrains) [reply WTWSMS/localisation/AO_events.csv]: _short is for targetted decision. I believe _short and _title have different usage

File: WTWSMS/events/WtWSMS_story_events.txt (hunk 7 above)
Comment (IhateTrains) [reply WTWSMS/events/WtWSMS_story_events.txt]: But you were talking about expanding this event's localisation, so I wanted to make it future-proof

File: WTWSMS/events/WtWSMS_story_events.txt (hunk 6 above)
Comment (IhateTrains) [reply WTWSMS/events/WtWSMS_story_events.txt]: That's right, I can remove it

File: WTWSMS/events/WtWSMS_story_events.txt (hunk 7 above)
Comment (loup99) [reply WTWSMS/events/WtWSMS_story_events.txt]: I already did that. And again, Root will always be used.

File: WTWSMS/localisation/AO_events.csv (hunk 4 above)
Comment (loup99) [reply WTWSMS/localisation/AO_events.csv]: Then it should be ROOT no?

File: WTWSMS/history/characters/hebrew.txt (hunk 8 above)
Comment (IhateTrains) [reply WTWSMS/history/characters/hebrew.txt]: I don't think there's any risk in this case :)

File: WTWSMS/events/WtWSMS_story_events.txt (hunk 6 above)
Comment (IhateTrains) [reply WTWSMS/events/WtWSMS_story_events.txt]: Done, changed to regular event_target

File: WTWSMS/events/WtWSMS_story_events.txt (hunk 7 above)
Comment (IhateTrains) [reply WTWSMS/events/WtWSMS_story_events.txt]: Done, removed.

File: WTWSMS/localisation/WtWSMS_events.csv (hunk 9 above)
Comment (IhateTrains) [reply WTWSMS/localisation/WtWSMS_events.csv]: Done, replaced with "dungeon".

File: WTWSMS/localisation/AO_events.csv (hunk 4 above)
Comment (IhateTrains) [reply WTWSMS/localisation/AO_events.csv]: Why Root?

Comment (loup99) [reply WTWSMS/localisation/AO_events.csv]: Because ROOT is the the targeted character in targeted decisions.

Comment (IhateTrains) [reply WTWSMS/localisation/AO_events.csv]: That could work then, I will test.

Comment (IhateTrains) [reply WTWSMS/localisation/AO_events.csv]: It still returned the decision taker...

Comment (loup99) [reply WTWSMS/localisation/AO_events.csv]: If the scopes are different in plots use FROM.

Comment (IhateTrains) [reply WTWSMS/localisation/AO_events.csv]: From didn't work too.

**Comment by IhateTrains** [comment]
I've added a dynasty for Samaritan High Priests :)

File: WTWSMS/history/characters/roman.txt [hunk 10]
```diff
@@ -1985,17 +1985,13 @@
 	420.1.1 = {
 		birth="420.1.1"
 	}
-	492.1.1 = {
-	name="Gelasius"
-	# AKA: Acacius
-	}
```
Comment (loup99) [inline WTWSMS/history/characters/roman.txt]: What happened here?

File: WTWSMS/history/characters/roman.txt [hunk 11]
```diff
@@ -1985,17 +1985,13 @@
...
 	496.11.21 = {
 		death="496.11.21"
 	}
 }
 
 7769 = {
-	name="Acacius"
+	name="Felix Anicius"
```
Comment (loup99) [inline WTWSMS/history/characters/roman.txt]: ❓

File: WTWSMS/localisation/AO_events.csv (hunk 4 above)
Comment (loup99) [reply WTWSMS/localisation/AO_events.csv]: What worked then? Those are the only scopes, all other plots use them...

File: WTWSMS/history/characters/roman.txt (hunk 10 above)
Comment (IhateTrains) [reply WTWSMS/history/characters/roman.txt]: I removed it on purpose. Was his birth name really Acacius?

File: WTWSMS/history/characters/roman.txt (hunk 11 above)
Comment (IhateTrains) [reply WTWSMS/history/characters/roman.txt]: Similar to above.

File: WTWSMS/localisation/AO_events.csv (hunk 4 above)
Comment (IhateTrains) [reply WTWSMS/localisation/AO_events.csv]: I couldn't find a working scope, so I used "Plot to Kidnap" instead.

File: WTWSMS/history/characters/roman.txt (hunk 10 above)
Comment (loup99) [reply WTWSMS/history/characters/roman.txt]: If that is was we have, presumably yes.

File: WTWSMS/history/characters/roman.txt (hunk 11 above)
Comment (loup99) [reply WTWSMS/history/characters/roman.txt]: Same as above.

File: WTWSMS/history/characters/roman.txt (hunk 10 above)
Comment (IhateTrains) [reply WTWSMS/history/characters/roman.txt]: I don't think it's an effect of deep research. Gelasius didn't change his name when he became Pope, his name was Gelasius... Maybe someone used Acacius character as a base?

Comment (IhateTrains) [reply WTWSMS/history/characters/roman.txt]: These Popes are outside of CK2 timeline, so PDX didn't really care much about them.

File: WTWSMS/history/characters/roman.txt (hunk 11 above)
Comment (loup99) [reply WTWSMS/history/characters/roman.txt]: Is Anicius in reference to the dynasty? Then that should not be included like that.

File: WTWSMS/history/characters/roman.txt (hunk 10 above)
Comment (loup99) [reply WTWSMS/history/characters/roman.txt]: If I'm not wrong, the mod adds some Popes too.

File: WTWSMS/history/characters/roman.txt (hunk 11 above)
Comment (IhateTrains) [reply WTWSMS/history/characters/roman.txt]: It was part of his name, but now I think know why it may look bad in game (Felix Anicius Anicius) :P I will remove the "Anicius" part

File: WTWSMS/history/characters/roman.txt (hunk 10 above)
Comment (IhateTrains) [reply WTWSMS/history/characters/roman.txt]: This Pope is from vanilla, I checked

File: WTWSMS/history/characters/scottish.txt [hunk 12]
```diff
@@ -52,7 +52,7 @@
 	intrigue=9
 	stewardship=9
 	religion="nicene"
-	culture="scottish"
+	culture=levantine_arabic
```
Comment (loup99) [inline WTWSMS/history/characters/scottish.txt]: Arabic? Isn't that anachronistic here?

Comment (IhateTrains) [reply WTWSMS/history/characters/scottish.txt]: This is what some other Apostles use. St. Andrew was born in a Jewish family though. What culture represents the Jews from Galilee?

Comment (IhateTrains) [reply WTWSMS/history/characters/scottish.txt]: Hebrew maybe?

Comment (loup99) [reply WTWSMS/history/characters/scottish.txt]: Hebrew probably.

Comment (IhateTrains) [reply WTWSMS/history/characters/scottish.txt]: Done, changed to hebrew
